group_vars/dataspace/production.yml

```diff
@@ -0,0 +1,68 @@
+---
+dspace_install_path: "/mnt/disks/{{ project }}{{ env }}datastore/prod-copy"
+dspace_install_directory: "{{ dspace_install_path }}/dspace"
+env: "prod"
+project: "dataspace"
+nginx_type: opensource
+nginx_install_from: nginx_repository
+deploy_user: "dspace"
+deploy_user_uid: 1005
+java_type: jdk
+tomcat_user: "{{ deploy_user }}"
+tomcat_group: "{{ deploy_user }}"
+tomcat_install_path: "/home/{{ deploy_user }}"
+dspace_db_user: 'dataspace_prod_db_user'
+dspace_db_password: "{{ vault_dataspace_production_db_password }}"
+dspace_db_name: "dataspace_prod_db"
+dspace_db_port: "5432"
+
+datadog_api_key: "{{ vault_datadog_key }}"
+datadog_enabled: true
+datadog_config:
+  log_enabled: true
+datadog_checks:
+  dataspace_log_check:
+    init_config:
+    logs:
+      - type: file
+        path: /dspace/log/dspace.log
+        service: dspace
+        source: dspace
+        sourcecategory: dspace_web_access
+        tags: "application:dataspace, environment:production, type:dspace"
+
+maven_version: 3.6.3
+ant_version: 1.10.8
+db_clusteradmin_password: "{{ vault_postgres_admin_password }}"
+db_clusteradmin_user: "postgres"
+
+postgres_port: "{{ dspace_db_port }}"
+postgres_admin_user: "{{ db_clusteradmin_user }}"
+postgres_admin_password: "{{ db_clusteradmin_password }}"
+postgresql_is_local: false
+postgres_version: 12
+postgres_host: "gcp-postgres-prod1"
+atmire_git_path: "https://{{ atmire_username|urlencode}}:{{ atmire_password|urlencode }}@{{ atmire_git_repo }}"
+atmire_dspace_git_path: "https://{{ atmire_username|urlencode}}:{{ atmire_password|urlencode }}@{{ atmire_dspace_git_repo }}"
+atmire_git_repo: "{{ vault_atmire_git_repo }}"
+atmire_dspace_git_repo: "{{ vault_atmire_dspace_git_repo }}"
+atmire_git_home: "{{ vault_atmire_git_home }}"
+atmire_git_branch: "dataspace-prod"
+atmire_path: "{{ vault_atmire_path }}"
+atmire_username: "{{ vault_atmire_username }}"
+atmire_password: "{{ vault_atmire_password }}"
+dspace_app_env: "{{ env }}"
+dspace_app_name: "dataspace"
+dspace_app_vars:
+  - name: PGHOST
+    value: "{{ postgres_host }}"
+  - name: PGDATABASE
+    value: "{{ dspace_db_name }}"
+  - name: PGUSER
+    value: "{{ dspace_db_user }}"
+  - name: PGPORT
+    value: "{{ dspace_db_port }}"
+  - name: PGPASSWORD
+    value: "{{ dspace_db_password }}"
+shib_hostname: "dataspace.pulcloud.io"
```

Comment:
This will need to be updated following the final production release, correct?

Reply:
that will be a separate PR